.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -419,6 +430,14 @@ async function main({ github, context }) {
 
   const report = generateReport({ issuesData, previousReport, componentData })
 
+  // if the ACT environment variable is set we are testing the action with
+  // https://nektosact.com/introduction.html
+  if (process.env.ACT) {
+    debug({msg: "printing out report..."})
+    console.log(report)
+    return
+  }
+
```

Comment:
Not opposed to this entirely, but I think this and the related discussion belongs on a separate PR.

Reply:
Happy to move it to a different PR, this is helpful when adding new functionalities to the report imho

Reply:
Please do! I agree that it is helpful but would prefer discussion on what tool is the best to use here and we can do that independently of this change

Reply:
done